src/test/scala/edu/ie3/simona/model/participant/ChpModelSpec.scala

```diff
@@ -190,26 +190,28 @@ class ChpModelSpec
     "Check total energy after calculating next state with #chpState and heat demand #heatDemand kWh:" in {
       val testCases = Table(
         ("chpState", "storageLvl", "heatDemand", "expectedTotalEnergy"),
-        (chpStateNotRunning, 90, 0, 0), // tests case (false, false, true)
+        (chpStateNotRunning, 70, 0, 0), // tests case (false, false, true)
+        // Fixme Check this, why 100?
```

Comment:
```suggestion
```

Like I said, a fresh view the next day make things easier: 50 kW for TWO hours = 100 kWh :)